jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```
@@ -419,9 +433,9 @@ private static String startLocalAgent() throws IbmAttachOperationFailedException
 			}
 		}
 		if (Objects.isNull(addr)) {
-			/* startLocalAgent() should have set the property. */
+			/* property should be set at this point */
 			IPC.logMessage(LOCAL_CONNECTOR_ADDRESS + " not set"); //$NON-NLS-1$
-			throw new IbmAttachOperationFailedException("startLocalManagementAgent: " + LOCAL_CONNECTOR_ADDRESS + " not defined"); //$NON-NLS-1$ //$NON-NLS-2$
+			throw new IbmAttachOperationFailedException(LOCAL_CONNECTOR_ADDRESS + " not defined"); //$NON-NLS-1$
```

Does this need to throw an exception? Seems it could just return null.